draft-ietf-tokbind-https-08.xml

```diff
@@ -486,18 +489,18 @@
 	  Token Consumer, not just the client to which it was
 	  issued.</t>
 	  <t>Therefore, in the previous step, the Token Provider may
-	  want to include in the token the Token-Binding public key that the client
+	  want to include in the token the Token Binding ID that the client
     uses when communicating with the Token Consumer, thus 
-    <spanx style="emph">binding</spanx> the token to client's Token-Binding
-    key pair. The client
+    <spanx style="emph">binding</spanx> the token to the client's Token Binding
+    private key. The client
```

Comment:
Did a quick review of this and had one (possibly confused) comment. But otherwise think it looks pretty good.